### PR TITLE
feat: initial exchanges page

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/861mruakp

TODO:

- [x] Tests
- [x] Mobile Layout (currently uses standard mobile table formatting)
- [x] Placeholder Logo
- [x] Dropdowns
- [x] Currency Formatting
- [ ] ~Handle "upto 4 decimals" as it currently shows 4 DP regardless~

Requires https://github.com/ArkEcosystem/laravel-foundation/pull/562 & https://github.com/ArkEcosystem/laravel-foundation/pull/563

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -69,4 +70,10 @@
...
 
 Route::view('/compatible-wallets', 'app.compatible-wallets')->name('compatible-wallets');
-Route::view('/exchanges', 'app.compatible-wallets')->name('exchanges');
+Route::get('/exchanges', function () {
+    if (! Network::canBeExchanged()) {
+        return redirect()->route('home');
+    }
+
+    return view('app.exchanges');
+})->name('exchanges');
```
Comment (alfonsobries) [inline routes/web.php:79]: I think we should move this to a controller

**Review by alfonsobries — APPROVED** [review]
im approving since my only comment is more like a suggestion

**Comment by alfonsobries** [comment]
corners are cut off 
image

**Comment by alfonsobries** [comment]
border on icons is missing 

Design:
image

Implementation
image

**Comment by alfonsobries** [comment]
On sm creen the submit wallet button is below

Design:
image


Implementation
image

**Comment by alfonsobries** [comment]
also card header color is not correct, is blueish in design

Design

image


Implementation
image

**Comment by alfonsobries** [comment]
Md screen does not show the volume

Design
image


Implementation
image

**Comment by alfonsobries** [comment]
Button says "submit wallet" but should say "submit exchange"

image

**Comment by alfonsobries** [comment]
lg screen (1024) is broken

Design:

image


Implementation

image

**Comment by alfonsobries** [comment]
margin above blue box should be 32px and is 24px 
image

**Review by alfonsobries — CHANGES_REQUESTED** [review]
design needs adjustments

**Comment by alexbarnsley** [comment]
@alfonsobries Fixed all the points apart from:

- lg screen (1024) is broken - **this is because there's no `md-lg` breakpoint yet - added here https://github.com/ArkEcosystem/laravel-foundation/pull/568**
- Md screen does not show the volume - **Same as above**
- Button says "submit wallet" but should say "submit exchange" - **Not implemented yet - https://app.clickup.com/t/861mrubj4**
- also card header color is not correct, is blueish in design - **that's a mistake in the designs, I confirmed with Oleg that it should be `gray-100`**

I'll re-open the PR once the foundation change is merged

**Review by samharperpittam — CHANGES_REQUESTED** [review]
The table styling at 640 is incorrect.

Design
image


Imp
image

**Review by samharperpittam — CHANGES_REQUESTED** [review]
The dropdown boxes are 4px too big in height. Should be 44px.

image

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**